src/App.css

```diff
@@ -1,5 +1,286 @@
-* {
+*,
```

Comment:
As you can see, your design has a couple of issues and does not resemble the given guidelines. Please be sure to make the necessary changes so that your project meets the requirements 🔥

Your design 
![image](https://user-images.githubusercontent.com/15807343/188290597-4692321a-eb29-474c-adc2-b956dd49a695.png)

Given design 
![image](https://user-images.githubusercontent.com/15807343/188290605-33aeb481-993d-4396-a871-ae521c4f235a.png)

Kindly work your margins and background colors. Also, your user icon is misplaced. Work on these issues and you should be good to go!